src/coreclr/jit/gentree.cpp

```diff
@@ -11702,6 +11702,66 @@ GenTree* Compiler::gtFoldExprCall(GenTreeCall* call)
     return call;
 }
 
+#ifdef FEATURE_HW_INTRINSICS
+
+//------------------------------------------------------------------------
+// gtFoldHWIntrinsic: see if a HWIntrinsic is foldable
+//
+// Arguments:
+//    node - HWIntrinsic node to examine
+//
+// Returns:
+//    The original node if no folding happened.
+//    An alternative tree if folding happens.
+//
+// Notes:
+//    Checks for HWIntrinsic nodes to Vector64.Create/Vector128.Create/Vector256.Create,
+//    and if the call is to one of these, attempt to optimize.
+
+GenTree* Compiler::gtFoldHWIntrinsic(GenTreeHWIntrinsic* node)
+{
+    // Defer folding if not optimizing.
```

Comment:
"Defer" perhaps not the right terminology (since we're not going to retry later in phase order)?

To be honest, the code is rather clear, not sure if there's a need to comment on it.

Reply:
That's fair, we are not doing this in a later phase. I think we can just get rid of the comment.